app/services/email_service/ses/client.rb

```diff
@@ -88,6 +88,28 @@ def set_notification_topic(email:, type:)
       end
     end
 
+    def disable_email_forwarding(email:)
+      if email.blank?
+        raise ArgumentError.new("Missing mandatory value for email parameter.")
+      end
+
+      method_params = {identity: email, forwarding_enabled: false}
+
+      log_request_response(:set_identity_feedback_forwarding_enabled, method_params) do
+        begin
+          response = @ses.set_identity_feedback_forwarding_enabled(method_params)
+          if response.successful?
+            Result::Success.new()
+          else
+            Result::Error.new(response.error)
+          end
+        rescue StandardError => e
+          Result::Error.new(e)
+        end
```

Comment:
There's quite a lot of repetition. This is the 4th time this pattern is present in this file: log, do request, if success, if error, catch exception.

I don't know, maybe we could add a helper or something. On the other it might be that adding a helper would actually make the code harder to read and understand.